play/apps/tournament/models/tournament_test.py

```diff
@@ -28,83 +28,86 @@ def _arrange(num_snakes=10):
         snakes.append(snake)
     return snakes, tournament, tournament_brackets
 
-
```

Comment:
I'm confused about what's going on here?

Reply:
we changed the model that's being used to associate the tournament/bracket/snake, and these tests needed updated. I commented them out and forgot to update them (I'll do that now, sorry for the preemptive pr)